lib/features/resize/ResizeHandles.js

```diff
@@ -172,7 +174,7 @@ ResizeHandles.prototype.removeResizers = function() {
 };
 
 ResizeHandles.prototype._getResizersParent = function() {
-  return this._canvas.getLayer('resizers');
+  return this._canvas.getLayer('resizers', LAYER_INDEX);
```

Comment:
I don't like that change as it essentially means all special layers must explicitly render on top of the base layer.

What about the following:

* if no priority is given, layers render with priority 1
* base layers + plane layers render with the priority of `0`

Please keep the canvas spec around. We should aim to fix this as part of our core.

Reply:
> I don't like that change as it essentially means all special layers must explicitly render on top of the base layer.

This means: Breaking change for everyone who created layers in the past, bpmn-js, some extension we don't yet know about...

Reply:
Makes sense, we don't want a major release for this :+1: